feat: [dlnfs] dlnfs support in vfat\exfat\ntfs

Lighto-Ku opened the pull request:
allow files renamed to long name in vfat\exfat\ntfs filesystem.

Log: dlnfs support in vfat\exfat\ntfs
Task: https://pms.uniontech.com/task-view-327271.html

Lighto-Ku committed:
feat: [dlnfs] dlnfs support in vfat\exfat\ntfs

allow files renamed to long name in vfat\exfat\ntfs filesystem.

Log: dlnfs support in vfat\exfat\ntfs
Task: https://pms.uniontech.com/task-view-327271.html

max-lvs commented on src/dfm-base/utils/fileutils.cpp:
```diff
@@ -1312,6 +1312,19 @@ QString FileUtils::normalPathToTrash(const QString &normal)
     return trash;
 }
 
+bool FileUtils::supportRenameToLongName(const QUrl &url)
```
bool FileUtils::supportRenameToLongName(const QUrl &url)  函数名修改为 supportLongName

Lighto-Ku committed:
feat: [organizer] unsupport large and super large icon size

1.unsupport large and super large icon size
2.add space in collect widgets

Log: organizer feature
Task: https://pms.uniontech.com/task-view-329805.html

Lighto-Ku committed:
fix: [workspace] tool tips issue

show tool tips in tree view mode.

Log: fix display issue
Bug: https://pms.uniontech.com/bug-view-246959.html

Lighto-Ku commented:
/merge
